NGit/NGit.Patch/BinaryHunk.cs

```diff
@@ -168,7 +168,7 @@ internal virtual int ParseHunk(int ptr, int end)
 			//
 			while (ptr < end)
 			{
-				bool empty = buf[ptr] == '\n';
+				bool empty = (buf[ptr] == '\n') || (buf[ptr] == '\r' && buf[ptr-1] == '\n');
```

Comment:
The test does not cover this change, so appears to be unnecessary. Why are we doing it? If it is important, can we please add a test for this?